docs/basic-features/script.md

```diff
@@ -123,6 +124,119 @@ Examples of scripts that do not need to load immediately and can be lazy-loaded
+
+```bash
+npm run dev
+
+# You'll see instructions like these:
+#
+# Please install Partytown by running:
+#
+#         npm install @builder.io/partytown
+#
+# ...
+```
+
+Once setup is complete, defining `strategy="worker` will automatically instantiate Partytown in your application and off-load the script to a web worker.
+
+```jsx
+<Script src="https://example.com/analytics.js" strategy="worker" />
+```
+
+There are a number of trade-offs that need to be considered when loading a third-party script in a web worker. Please see Partytown's [Trade-Offs](https://partytown.builder.io/trade-offs) documentation for more information.
+
+#### Configuration
+
+Although the `worker` strategy does not require any additional configuration to work, Partytown supports the use of a config object to modify some of its settings, including enabling `debug` mode and forwarding events and triggers.
+
+If you would like to add additonal configuration options, you can include it within the `<Head />` component used in a [custom `_document.js`](/docs/advanced-features/custom-document.md):
+
+```jsx
+import Document, { Html, Head, Main, NextScript } from 'next/document'
+
+class MyDocument extends Document {
+  render() {
+    return (
+      <Html>
+        <Head>
+          <script
+            data-partytown-config
+            dangerouslySetInnerHTML={{
+              __html: `
+              partytown = {
+                lib: "/_next/static/~partytown/",
+                debug: true
+              };
+            `,
+            }}
+          />
+        </Head>
+        <body>
+          <Main />
+          <NextScript />
+        </body>
+      </Html>
+    )
+  }
+}
+
+export default MyDocument
+```
+
+In order to modify Partytown's configuration, the following conditions must be met:
+
+1. The `data-partytown-config` attribute must be used in order to overwrite the default configuration used by Next.js
+2. Unless you decide to save Partytown's library files in a separate directory, the `lib: "/_next/static/~partytown/"` property and value must be included in the configuration object in order to let Partytown know where Next.js stores the necessary static files.
+
+> **Note**: If you are using an [asset prefix](/docs/api-reference/next.config.js/cdn-support-with-asset-prefix.md) and would like to modify Partytown's default configuration, you must include it as part of the `lib` path:
+>
+> ```jsx
+> import { assetPrefix } from '../next.config'
+>
+> class MyDocument extends Document {
+>   render() {
+>     return (
+>       <Html>
+>         <Head>
+>           <script
+>             data-partytown-config=""
+>             dangerouslySetInnerHTML={{
+>               __html: `
+>               partytown = {
+>                 lib: "${assetPrefix}/_next/static/~partytown/"
+>               };
+>             `,
+>             }}
+>           />
+>         </Head>
+>         // ...
+>       </Html>
+>     )
+>   }
+> }
+>
+> export default MyDocument
+> ```
```

Comment:
```suggestion
```
I think we can omit this snippet and the above note should be sufficient, not sure we want to encourage importing directly from `next.config.js` as this could pull in dev dependencies

Reply:
Updated! (and TIL it may not be good practice to import from `next.config.js` :))